src/server/snap.rs

```diff
@@ -154,8 +151,10 @@ fn send_snap(
     };
 
     let cb = ChannelBuilder::new(env)
-        .keepalive_time(Duration::from_secs(60))
-        .keepalive_timeout(Duration::from_secs(3));
+        .stream_initial_window_size(cfg.grpc_stream_initial_window_size.0 as usize)
```

Comment:
It may overflow.

Reply:
Yes. Just like all other similar places in the codebase, it should be fixed after grpcio is updated.